src/content/administration/dashboard.mdx

```diff
@@ -177,12 +177,16 @@ From this page you can follow a simple step-by-step guide to get the major compo
 
 ## Registry Credentials
 
-This section in the admin view allows you to see the registry credentials that have been [configured through the helm chart](/self-hosted/administration/configuration/#privateregistry). You will see a row for each configured registry with the type (Static or AWS) and the registry hostname. The 2 types of credentials supported are:
-- Static: To configure registries with static username and password like Docker Hub.
-- AWS: To configure [ECR registries](https://docs.aws.amazon.com/ecr/index.html) using Access Key and Access Secret. In this case, Okteto takes care of refreshing docker credentials every 4 hours using the key and secret configured.
+This section of the Admin dashboard lets you manage credentials for private registries to use in your Okteto instance. The credentials you specify here are injected by Okteto into any operation (e.g. build, deploy) you perform (UI or CLI) so developers don't need access to them. You only need to configure your credentials here and all developers in your Okteto instance will be able to interact with those registries while developing with Okteto. If the setting [`daemonset.configurePrivateRegistriesInNodes.enabled`](/self-hosted/administration/configuration.mdx/#daemonset) is enabled, Okteto's daemonset will also configure these credentials in the nodes so the cluster can pull images from those registries.
```

Comment:
```suggestion
In this section of the Admin dashboard you can manage the private registry credentials for your Okteto instance. The credentials you configure here are injected by Okteto into any operation (e.g. build, deploy) performed by developers so they don't need direct access to the credentials. Once you configure the credentials here all developers in your Okteto instance will be able to use the respective registries while developing with Okteto. 

If the setting [`daemonset.configurePrivateRegistriesInNodes.enabled`](/self-hosted/administration/configuration.mdx/#daemonset) is enabled, Okteto's daemonset will also configure these credentials in the nodes so the cluster can also pull images from those registries.
```

I took a stab at breaking this up a bit to make it a bit easier to digest and also make it a bit more concise